lms/djangoapps/teams/static/teams/js/collections/topic.js

```diff
@@ -25,9 +25,7 @@
                 },
 
                 onUpdate: function(event) {
-                    if (event.action === 'create') {
-                        this.isStale = true;
-                    }
+                    this.isStale = this.isStale || _.contains(['create', 'delete'], event.action);
```

Comment:
Nit: I'm finding this hard to read. How about:

    if (_.contains(['create', 'delete'], event.action)) {
        this.isStale = true;
    }

I think that better captures the logic, which is that we only mark the collection as stale for events of that action type.